controller.py

```diff
@@ -372,10 +374,16 @@ def run_encoding(
             request.project_id,
             f"embedding:{embedding_id}:state:{enums.EmbeddingState.FINISHED.value}",
         )
+        notification_msg = (
+            f"Finished encoding {attribute_name} using model {request.config_string}."
+        )
+        if len(embedder.get_warnings()):
+            notification_msg += " Warnings: " + embedder.get_warnings()[-1]
+
```

Comment:
i'd create an own notification for this since SUCCESS and WARNING are two different states

Reply:
good suggestion, i will create an own notification